src/main/java/Library/Main.java

```diff
@@ -1,25 +1,48 @@
 package Library;
 
-import Data.BookHandler;
+import Data.UserManager;
 import Interface.Menu;
 
+import java.util.InputMismatchException;
+
 public class Main {
     public static void main(String[] args) {
+
+        UserManager userManager = new UserManager();
+
         Library library = new Library();
         Menu menu = new Menu();
```

Comment:
Remove if this is unused

Reply:
It is actually used in :if (userManager.login(Menu.getUserReply("Please enter your username.")."
I kept it as it was.